File: cpp_e9zMZ/cpp/06-thaphinh/17.cpp
#include <iostream>

#include "zvn.h"
using namespace std;

int main() {
    int height = 7;
    string shape = "";

    for (int i = 1; i <= height; i++) {
        string charater = myRepeat("#", height - i);
        string left = createListNumber(1, i - 1);
        string right = createListNumberReverse(i, 1);
        string record = charater + left + right + charater;
        shape += record + "\n";
    }
    
    cout << shape << "\n";
}

File: python_ToSXs/python/06-thaphinh/17.py
height = 8

shape = ""
for i in range(1, height + 1):
    around = "#" * (height - i)

    left = [str(j) for j in range(1, i)]
    left = "".join(left)
    right = [str(j) for j in range(i - 1, 0, -1)]
    right = "".join(right)

    center = left + str(i) + right
    record = around + center + around

    shape += record + "\n"

print(shape)
